mod/config/pkg/spec/special_cases.go

```diff
@@ -28,3 +30,9 @@ const (
 	//nolint:lll // temporary.
 	BArtioValRoot = "0x9147586693b6e8faa837715c0f3071c2000045b54233901c2e7871b15872bc43"
 )
+
+const (
```

Comment:
```suggestion
// Planned hard-fork upgrades on boonet.
const (
```
or something similar